cli/templatecreate.go

```diff
@@ -46,6 +42,15 @@ func (r *RootCmd) templateCreate() *clibase.Cmd {
 			r.InitClient(client),
 		),
 		Handler: func(inv *clibase.Invocation) error {
+			_, _ = fmt.Fprintln(inv.Stdout, "\n"+pretty.Sprint(cliui.DefaultStyles.Wrap,
```

Comment:
I'm not super familiar with the API of `clibase` but I think this would be better off implemented there so we can reuse it...maybe as a middleware?

I understand the intent of the `time.Sleep` but I think we should either make it something that users have to explicitly acknowledge via `stdin` (and thus are able to opt out of via `-y`) or we just accept that it's possible for the output to missed.

Reply:
Will look into the middleware idea

Yeah Ammar really liked the idea of the sleep, I'm fine removing it because I don't think it's worth forcing an input from the user.